package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@across-protocol/constants",
-  "version": "3.1.10",
+  "version": "3.1.11",
```

We never produced a 3.1.10 so you're OK to reuse the existing version.

```suggestion
  "version": "3.1.10",
```